ft-sdk/src/data.rs

```diff
@@ -3,6 +3,7 @@ pub type Result = std::result::Result<ft_sdk::chr::CHR<Output>, ft_sdk::Error>;
 #[derive(Debug)]
 pub enum Output {
     Json(serde_json::Value),
+    Zip((String, bytes::Bytes)),
```

Comment:
Why zip? This can be any binary data with arbitrary content type. Create Binary and accept bytes, content type.